src/main/java/uk/gov/companieshouse/service/dissolution/DissolutionEmailService.java

```diff
@@ -64,6 +66,11 @@ public void sendApplicationOutcomeEmail(Dissolution dissolution, DissolutionVerd
                 this.getApplicationAcceptedEmailDocument(dissolution) :
                 this.getApplicationRejectedEmailDocument(dissolution, dissolutionVerdict);
 
+        if (dissolutionVerdict.getResult() == VerdictResult.REJECTED) {
+            EmailDocument<?> financeEmailCopy = this.getApplicationRejectedEmailDocumentForFinance(dissolution, dissolutionVerdict);
```

Comment:
You set `emailDocument` above when the verdict is rejected, and then you overwrite it here.
Also if you make the change I mentioned above, we can reuse the `getApplicationRejectedEmailDocument` private method, and pass in the two separate email address.

So this could look something like this
```
public void sendApplicationOutcomeEmail(Dissolution dissolution, DissolutionVerdict dissolutionVerdict) {
    if (dissolutionVerdict.getResult() == VerdictResult.ACCEPTED) {
        EmailDocument<?> applicationAcceptedEmailDocument = this.getApplicationAcceptedEmailDocument(dissolution);
        sendEmail(applicationAcceptedEmailDocument);
    } else {
        EmailDocument<?> applicationRejectedEmailDocument = this.getApplicationRejectedEmailDocument(dissolution, dissolutionVerdict, dissolution.getCreatedBy().getEmail());
        sendEmail(applicationRejectedEmailDocument);

        EmailDocument<?> applicationRejectedEmailDocumentForFinanceTeam = this.getApplicationRejectedEmailDocument(dissolution, dissolutionVerdict, envConfig.getChsFinanceEmail());
        sendEmail(applicationRejectedEmailDocumentForFinanceTeam);
    }
}
```